test/benchmark/simulator_benchmark.py

```diff
@@ -118,28 +114,28 @@ def __init__(self,
             self.noise_models[self.NOISE_DEPOLARIZING] = noise_model
 
         if self.RUNTIME_STATEVECTOR_CPU in runtime_names:
-            self.simulators[self.RUNTIME_STATEVECTOR_CPU] = SIMULATOR
-            self.backend_options_list[self.RUNTIME_STATEVECTOR_CPU] = { 'method': self.RUNTIME_STATEVECTOR_CPU }
+            self.simulators[self.RUNTIME_STATEVECTOR_CPU] = AerSimulator(method='statevector', device='CPU')
+            self.backend_options_list[self.RUNTIME_STATEVECTOR_CPU] = { 'enable_truncation': False}
```

Comment:
I think with all these removals, the `SIMULATOR` module constant can now be removed, since it's only used by `all_simulators` which also isn't accessed?